src/plugins/walletActions/transaction.ts

```diff
@@ -79,70 +114,84 @@ interface WithdrawParams {
  * @param feeToken
  * @param amount
  * @param fastWithdraw
- * @param fees
+ * @param fee
  * @param store
  * @return {Promise<{txData: *, txHash: *}[]>}
  */
-export const withdraw = async ({ address, token, feeToken, amount, fastWithdraw, fees, store }: WithdrawParams): Promise<Transaction> => {
+export const withdraw = async ({ address, token, feeToken, amount, fastWithdraw, fee, accountActivationFee, store }: WithdrawParams) => {
   const syncWallet = walletData.get().syncWallet;
-  const amountBigValue = amount;
-  const feeBigValue = fees;
+  const nonce = await syncWallet!.getNonce("committed");
+  const batchBuilder = syncWallet!.batchBuilder(nonce);
+
+  if (store.wallet.isAccountLocked) {
+    let pubKeyTx = window.localStorage.getItem(`pubKeySignature-${store.account.address}`);
```

Comment:
Maybe create a separate function for getting the signature from the localStorage to mitigate the possibility of typo?